include/NovelRT/Maths/Utilities.h

```diff
@@ -94,27 +142,81 @@ namespace NovelRT::Maths::Utilities
         }
     }
 
+    /**
+     * @brief Converts an angle in degrees to an angle in radians.
+     *
+     * @details
+     * To get the angle in radians you can use this formula: \f[
+     *      radians = \frac{\pi \times degrees}{180}
+     * \f]
+     *
+     * @param degrees The angle in degrees to convert.
+     * @return The angle in radians.
+     */
     [[nodiscard]] inline float DegreesToRadians(float degrees) noexcept
     {
         return glm::radians(degrees);
     }
 
+    /**
+     * @brief Converts an angle in degrees to an angle in radians.
+     *
+     * @details
+     * To get the angle in degrees you can use this formula: \f[
+     *      degrees = \frac{180 \times radians}{\pi }
+     * \f]
+     *
+     * @param radians The angle in radians to convert.
+     * @return The angle in degrees.
+     */
     [[nodiscard]] inline float RadiansToDegrees(float radians) noexcept
     {
         return glm::degrees(radians);
     }
 
+    /**
+     * @brief Retrieves a constant for a full rotation expressed in radians.
+     *
+     * @details
+     * A full rotation is expressed as: \f[
+     *      2\pi = 6.283185307179586476925...
+     * \f]
+     *
+     *
+     * @tparam TFloatingPointType A floating point type that is expressed as either float or double.
+     * @return A full rotation expressed in radians.
+     */
     template<typename TFloatingPointType>[[nodiscard]] inline constexpr TFloatingPointType Tau() noexcept
     {
         _assert(std::is_same_v<TFloatingPointType, float> || std::is_same_v<TFloatingPointType, double>,
                 "Type is unsupported for a Tau representation.");
     }
 
+    /**
+     * @brief Retrieves a constant for a full rotation expressed in radians.
+     *
+     * @details
+     * A full rotation is expressed as: \f[
+     *      2\pi = 6.283185307179586476925...
+     * \f]
```

Comment:
I'm not sure if this is correct given the limitations of our float implementation of `Tau`.